tests/h/session_test.py

```diff
@@ -69,8 +69,28 @@ def test_model_show_sidebar_tutorial(
         assert 'show_sidebar_tutorial' not in preferences
 
 
+def test_profile_userid_unauthenticated(unauthenticated_request):
```

Comment:
Yeah, usually parameterizing is the way to go, but since we know that the `model` tests will be removed over the long term I don't mind duplicating the tests. 👍